### PR TITLE
Add 0.13.2 docs

### PR DESCRIPTION
**Pull request by nastra**

### CODE REVIEW

File: docs/content/docs/docs/api/_index.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
```
Comment (samredai) [inline docs/content/docs/docs/api/_index.md:1]: These files shouldn't be in a nested `content/docs/docs` directory. All of the sections should be placed directly in `content/docs`. The file tree for `content` should look like this.
```
content
└── docs
    ├── api
    │   ├── _index.md
    │   ├── java-api-quickstart.md
    │   ├── java-api.md
    │   ├── java-custom-catalog.md
    │   ├── javadoc
    │   │   └── _index.md
    │   ├── python-api-intro.md
    │   ├── python-feature-support.md
    │   └── python-quickstart.md
    ├── asf
    │   ├── _index.md
    │   ├── donate
    │   │   └── _index.md
    │   ├── events
    │   │   └── _index.md
    │   ├── license
    │   │   └── _index.md
    │   ├── security
    │   │   └── _index.md
    │   └── sponsors
    │       └── _index.md
    ├── athena
    │   └── _index.md
    ├── community
    │   ├── _index.md
    │   ├── blogs.md
    │   ├── contributing.md
    │   ├── join.md
    │   └── talks.md
    ├── dremio
    │   └── _index.md
    ├── emr
    │   └── _index.md
    ├── flink
    │   ├── _index.md
    │   ├── flink-connector.md
    │   └── flink-getting-started.md
    ├── format
    │   ├── _index.md
    │   ├── spec.md
    │   ├── terms.md
    │   └── view-spec.md
    ├── hive
    │   └── _index.md
    ├── integrations
    │   ├── _index.md
    │   ├── aws.md
    │   ├── jdbc.md
    │   └── nessie.md
    ├── prestodb
    │   └── _index.md
    ├── project
    │   ├── _index.md
    │   ├── benchmarks.md
    │   ├── how-to-release.md
    │   ├── how-to-verify-a-release.md
    │   ├── multi-engine-support.md
    │   ├── roadmap.md
    │   ├── security.md
    │   └── trademarks.md
    ├── releases
    │   ├── 0.12.1
    │   │   └── _index.md
    │   ├── 0.13.0
    │   │   └── _index.md
    │   ├── 0.13.1
    │   │   └── _index.md
    │   ├── 0.13.2
    │   │   └── _index.md
    │   ├── _index.md
    │   ├── latest
    │   │   └── _index.md
    │   └── release-notes.md
    ├── spark
    │   ├── _index.md
    │   ├── spark-configuration.md
    │   ├── spark-ddl.md
    │   ├── spark-getting-started.md
    │   ├── spark-procedures.md
    │   ├── spark-queries.md
    │   ├── spark-structured-streaming.md
    │   └── spark-writes.md
    ├── starrocks
    │   └── _index.md
    ├── tables
    │   ├── _index.md
    │   ├── configuration.md
    │   ├── evolution.md
    │   ├── maintenance.md
    │   ├── partitioning.md
    │   ├── performance.md
    │   ├── reliability.md
    │   └── schemas.md
    └── trino
        └── _index.md
```

File: docs/content/docs/content/_index.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
+---
```
Comment (samredai) [inline docs/content/docs/content/_index.md:1]: This file should already exists in [docs/content/_index.md](https://github.com/apache/iceberg-docs/blob/main/docs/content/_index.md) and shouldn't need to be copied here. Changes can be made to the existing location and that will get updated as soon as the version branch is cut.

**Comment by samredai** [comment]
@nastra the extra `docs` directory is the root cause of the issues in the other comments. I'm realizing it's because this sentence in the instructions could be clearer and might have been interpreted as copying the directory :
> 3. Copy the contents under `docs` in the iceberg repo to `./docs/content/docs` in the **release** branch

That would be stated more clearly as:

> 3. Copy the `docs` directory in the iceberg repo into `./docs/content/` in the **release** branch

See the tree structure in [this](https://github.com/apache/iceberg-docs/pull/85#discussion_r889009930) comment. So for example the spark directory of markdown files should be `docs/content/docs/spark`, not `docs/content/docs/docs/spark`.

File: docs/content/docs/docs/api/_index.md (hunk 1 above)
Comment (nastra) [reply docs/content/docs/docs/api/_index.md]: should be fixed now

File: docs/content/docs/releases/0.13.1/_index.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+---
+title: "0.13.1"
```
[the remark this answers is not in the document]
Comment (nastra) [reply docs/content/docs/releases/0.13.1/_index.md:2]: ah you probably mean we need to copy over the stuff from https://github.com/apache/iceberg/pull/4954 ?

[the remark this answers is not in the document]
Comment (samredai) [reply docs/content/docs/releases/0.13.1/_index.md:2]: https://github.com/apache/iceberg/pull/4954

[the remark this answers is not in the document]
Comment (nastra) [reply docs/content/docs/releases/0.13.1/_index.md:2]: included the index file for 0.13.2

**Pull request merged**